.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,125 @@
+---
+name: Shellcheck
+on:
+  pull_request:
+env:
+  SHELLCHECK_REPO: 'koalaman/shellcheck'
+  SHELLCHECK_VERSION: 'v0.9.0'
+  SHELLCHECK_SHA: '038fd81de6b7e20cc651571362683853670cdc71'
```

Comment:
Are Renovate or Dependabot able to monitor new releases for this kind of definition?

Reply:
I suspect Renovate / Dependabot won't catch this, but thankfully shellcheck only seems to [release about once a year](https://github.com/koalaman/shellcheck/releases), and I don't think we'd miss out on much if we were one release behind.